datafusion/sqllogictest/test_files/errors.slt

```diff
@@ -119,13 +119,36 @@ regr_slope(c11, '2') over () as min1
 from aggregate_test_100
 order by c9
 
-# WindowFunction with BuiltInWindowFunction wrong signature
-statement error DataFusion error: Error during planning: No function matches the given name and argument types 'NTH_VALUE\(Int32, Int64, Int64\)'\. You might need to add explicit type casts\.\n\tCandidate functions:\n\tNTH_VALUE\(Any, Any\)
+# WindowFunction wrong signature
+statement error
 select
 c9,
 nth_value(c5, 2, 3) over (order by c9) as nv1
 from aggregate_test_100
 order by c9
+----
+DataFusion error: Error during planning: Error during planning: Coercion from [Int32, Int64, Int64] to the signature OneOf([Any(0), Any(1), Any(2)]) failed. No function matches the given name and argument types 'nth_value(Int32, Int64, Int64)'. You might need to add explicit type casts.
+	Candidate functions:
+	nth_value()
+	nth_value(Any)
+	nth_value(Any, Any)
```

Comment:
This format worked (for me) in local but it is failing in CI. The following change works (again for me) in local. I guess it will pass in CI as well.

```suggestion
statement error DataFusion error: Error during planning: Error during planning: Coercion from \[Int32, Int64, Int64\] to the signature OneOf\(\[Any\(0\), Any\(1\), Any\(2\)\]\) failed
select
c9,
nth_value(c5, 2, 3) over (order by c9) as nv1
from aggregate_test_100
order by c9
```

Reply:
Just trying to fix that. Thanks so much @jcsherin